src/components/AddPeopleAndRoles/OrgPerson.vue

```diff
@@ -1,56 +1,65 @@
 <template>
-  <div id="addEditPersonContainer">
-    <confirm-dialog ref="reassignCPDialog" attach="#addEditPersonContainer" />
+  <div id="add-edit-org-person">
```

Comment:
Kebab case for ids and classes. Also renamed for consistency with other components.